packages/cli-lib/api/marketplaceValidation.js

```diff
@@ -0,0 +1,45 @@
+const http = require('../http');
+
+const VALIDATION_API_BASE = 'quality-engine/v1/validation';
```

Comment:
OOC are these validation endpoints specific to themes?

Reply:
Yes, I believe so. @ssethia2, could you confirm?

Reply:
They are specific to themes, confirmed today.

Reply:
Should that be reflected in the naming - these read like generic marketplace validation endpoints.